plugin.video.ring_doorbell/ring_doorbell/const.py

```diff
@@ -0,0 +1,88 @@
+# coding: utf-8
+# vim:sw=4:ts=4:et:
+"""Constants."""
+import os
+from uuid import uuid4 as uuid
+
+HEADERS = {'Content-Type': 'application/x-www-form-urlencoded; charset: UTF-8',
+           'User-Agent': 'Dalvik/1.6.0 (Linux; Android 4.4.4; Build/KTU84Q)',
+           'Accept-Encoding': 'gzip, deflate'}
+
+# number of attempts to refresh token
+RETRY_TOKEN = 3
+
+# default suffix for session cache file
+CACHE_ATTRS = {'account': None, 'alerts': None, 'token': None}
+
+try:
+    CACHE_FILE = os.path.join(os.getenv("HOME"),
```

Comment:
I am afraid you can only write to the userdata of your addon. Can you please adapt?